src/elements/image/ImageElementForm.tsx

```diff
@@ -72,33 +74,37 @@ export const ImageElementForm: React.FunctionComponent<Props> = ({
           <FieldWrapper
             field={fields.altText}
             errors={errors.altText}
-            label="Alt text"
+            label={
+              <span>
+                <AltText>Alt text</AltText>
```

Comment:
Was this wrapper used to allow for customised styling?

Reply:
The `span`? No, it's not used to do anything other than wrap the content in a single parent – perhaps a fragment (`<> </>`) would be a better choice here?

Reply:
I think as long as we have a consistent pattern throughout all components? Im assuming both are inline tags in their behaviour.

Reply:
They are – but it's worth noting that `Fragment` in React doesn't actually resolve to a DOM node! A fragment is just a way of wrapping lists of DOM nodes to stop React complaining – it disappears in the final HTML:

```jsx
// This is what you're write in React
<>
  <div>Some</div>
  <div>list</div>
  <div>of</div>
  <div>things</div>
</>
```

```html
// This is what ends up on the page
<div>Some</div>
<div>list</div>
<div>of</div>
<div>things</div>
```

I'll change my implementation to use the fragment, the span isn't doing any useful work.